### ISSUE TITLE
[1-8] Price Target Settings (Buy/Sell/Stop-loss)

### ISSUE THREAD

**Issue by foxyberry** [foxyberry/quant-investment#17]
## Goal
Allow users to set price targets for each stock

## Tasks
- [ ] Define price target schema (buy_price, sell_price, stop_loss, take_profit)
- [ ] Implement save/load/update functions
- [ ] CLI interface for setting targets
- [ ] Validation (stop_loss < buy_price < take_profit)

## Acceptance Criteria
```python
set_price_targets(
    ticker="005930",
    buy=70000,
    sell=80000,
    stop_loss=65000,
    take_profit=90000
)

targets = get_price_targets("005930")
assert targets.buy == 70000
assert targets.stop_loss == 65000
assert targets.take_profit == 90000
```

## Technical Notes
- Support percentage-based targets (e.g., stop_loss = -7%)
- Store in `config/price_targets.yaml`
- Alert when current price approaches targets

## Parent
- #6 (Epic 1: Stock Discovery)

## Labels
`epic:stock-discovery`, `size:small`
